spec/registry.md

```diff
@@ -564,7 +564,7 @@ The following options and their values are required to be available on the funct
 - `numberingSystem`
    - valid [Unicode Number System Identifier](https://cldr-smoke.unicode.org/spec/main/ldml/tr35.html#UnicodeNumberSystemIdentifier)
      (default is locale-specific)
-- `currencySign`
+- `sign`
```

Comment:
It's not really "sign" in the usual sense, since the value `accounting` often means this sort of format: `($123.45)` and because it's easy to be confused about `sign` vs. `signDisplay`.

If we just want to get rid of the spurious `currency` in the name, I'd probably look to change `currencyDisplay` to `symbol` with options `auto` (or perhaps `regular`), `narrow`, `name`, `code`, `formal`, and `never`.

Reply:
Would `signDisplay` be better?

Reply:
We already have `signDisplay` with a long list of values, so it can't be that--unless we want to merge them?

Reply:
`symbol` sounds good.
The Unicode block is "Currency Symbols".

Wikipedia says
"A currency symbol or currency sign is a graphic symbol used..."
https://en.wikipedia.org/wiki/Currency_symbol

So both would be fine. But sign is more confusing.